.github/workflows/rpmbuild.yml

```diff
@@ -23,7 +23,7 @@ jobs:
 
       - name: generate linux tarball
         run: |
-          npm cache clear & npm install
+          make install
```

Comment:
Might need the `npm cache clear` step for stale audit issues?

Reply:
there are no previous dependencies installed in the container  for the rpmbuild workflow, I don't think there would be anything in the cache for us to clear here. Do we still see a need to add this?

Reply:
Makes sense, should be fine to remove